src/Tribe/Cache.php

```diff
@@ -180,6 +181,36 @@ public function delete_expired_transients() {
 				AND a.option_name NOT LIKE '\_transient\_timeout_tribe\_%'
 		";
 		$wpdb->query( $sql );
+
+		if ( wp_using_ext_object_cache() ) {
+			/**
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 3f378d445a32c0f030d0065c13b32196</sub>